src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -557,8 +558,7 @@ public void processOreRegistration(@Nullable Material material) {
                 PlasmaProperty plasmaProperty = material.getProperty(PropertyKey.PLASMA);
                 if (plasmaProperty != null && plasmaProperty.getPlasma() == null) {
                     int baseTemperature = fluidProperty == null ? 0 : fluidProperty.getFluidTemperature();
-                    baseTemperature = baseTemperature + 30000;
-                    Fluid fluid = MetaFluids.registerFluid(material, MetaFluids.FluidType.PLASMA, baseTemperature, false);
+                    Fluid fluid = MetaFluids.registerFluid(material, FluidTypes.PLASMA, baseTemperature + 30000, false);
```

Comment:
This change for temperature is incorrect, as it results in the incorrect temperature being added to the tooltip